springwolf-add-ons/springwolf-kotlinx-serialization-model-converter/src/main/kotlin/io/github/springwolf/addons/kotlinx_serialization_model_converter/converter/KotlinxSerializationModelConverter.kt

```diff
@@ -60,12 +61,18 @@ class KotlinxSerializationModelConverter(private val useFqn: Boolean = false) :
 
     private fun getPropertySchema(property: KProperty1<*, *>, context: ModelConverterContext): Schema<*> {
         val propertySchema: Schema<*>
-
-       val name = getPropertyName(property)
-
+        val name = getPropertyName(property)
```

Comment:
Seems like spotless is not configured to handle kotlin files as well

Could be a great future addition in https://github.com/springwolf/springwolf-core/blob/a3198a4cfbabe852c2ab425094ef478c649ab598/build.gradle#L54